components/team_general_tab/team_general_tab.tsx

```diff
@@ -275,15 +275,9 @@ export default class GeneralTab extends React.PureComponent {
         document.getElementById('team_settings')?.removeEventListener('hidden.bs.modal', this.handleClose);
     }
 
-    onOpenInviteToggle = (active) => {
-        if (active) {
-            this.handleUpdateSection('open_invite');
-        } else {
-            this.handleUpdateSection('');
-        }
-    }
+    onOpenInviteToggle = (active: boolean) => this.handleUpdateSection(active ? 'open_invite' : '');
```

Comment:
similar to the return type above comment

Reply:
same here 
<img width="735" alt="Screen Shot 2022-10-03 at 17 12 26" src="https://user-images.githubusercontent.com/37421564/193614428-94b70ea7-14f3-443b-adb5-8f3712b6f6eb.png">